RecoLocalTracker/SiPixelClusterizer/plugins/PixelThresholdClusterizer.cc

```diff
@@ -55,6 +55,10 @@ PixelThresholdClusterizer::PixelThresholdClusterizer
     theStackADC_( conf.exists("AdcFullScaleStack") ? conf.getParameter<int>("AdcFullScaleStack") : 255 ),
     theFirstStack_( conf.exists("FirstStackLayer") ? conf.getParameter<int>("FirstStackLayer") : 5 ),
     theElectronPerADCGain_( conf.exists("ElectronPerADCGain") ? conf.getParameter<double>("ElectronPerADCGain") : 135. ),
+    doPhase2Calibration( conf.exists("Phase2Calibration") ? conf.getParameter<bool>("Phase2Calibration") : false),
```

Comment:
Please add all these parameters to the fillDescription() method, and remove the check whether they exist.
Please do the same also for the ones above ("AdcFullScaleStack", "FirstStackLayer", "ElectronPerADCGain") even if not introduced by this PR